posts/effortless-data-ingestion-with-aws-glue.md

```diff
@@ -0,0 +1,56 @@
+
+AWS provides an encompassing Glue landscape where users can leverage technology to not only uncover data but also gain valuable insight to guide decisions and strategy. With numerous ways to leverage these capabilities, users from both technical and non-technical backgrounds can drive this effort forward. This article will discuss how to get started ingesting some common data streams with a non-technical approach, thus setting you on the course to make better-informed decisions for your business.
+
+# Common Input Streams
+Data resides in a myriad of locations. Below are a few popular data sources that can be leveraged in AWS Glue with no coding expertise needed.
+
+- **Databases**: Data is at the core of every business, and what better way to store that data than a database? AWS Glue is capable of connecting to multiple database engines, such as MySQL, PostgreSQL, DynamoDB, and many more.
+- **Amazon Simple Storage Service (S3)**: Amazon S3 is a common place for data storage and archiving. S3 is another popular data storage option with which Glue can integrate seamlessly.
+- **Snowflake**: A popular cloud-based data warehousing platform that provides powerful data storage, processing, and analysis capabilities. Coupled with AWS Glue, it can create a robust data integration solution.
+- **Apache Kafka**: A distributed streaming platform commonly built to process real-time event-driven data, representing another possible data ingestion point for AWS Glue.
+- **Kinesis**: A suite of AWS services that simplify the process of collecting and streaming data in real-time, all of which can be consumed by AWS Glue.
+  
+The examples above only capture a few of the possible data ingestion points that Glue can utilize. Be sure to check to see if Glue has an out-of-the-box connection for any of your data ingestion streams of interest.
+
+# Why Leverage Multiple Input Streams?
+Each piece of data represents a snippet of what makes your business special. By treating each input as a valuable source of information, AWS Glue can significantly enhance the flexibility, efficiency, and performance of your data processing capabilities. Utilizing multiple input streams will empower you to harness the full potential of your data, creating a versatile, efficient, and cost-effective data integration process that can adapt to the dynamic needs of your organization's complex data ecosystem.
+
+# Getting Started with AWS Glue Visual ETL
+AWS glue provides a visual interface designed for creating ETL jobs. This interface enables users to build ETL workflows utilizing a graphical representation of the data transformation process. These visually composed workflows can be run on AWS Glue’s Apache Spark-based ETL engine, allowing you to inspect the data each step of the way.
```

Comment:
Use expanded acronym first. i.e. Extract Transform Load (ETL) - then safe to use ETL after first definition.